packages/web-components/src/image/image.ts

```diff
@@ -0,0 +1,64 @@
+import { attr, FASTElement } from '@microsoft/fast-element';
+import { ImageBorderRadius, ImageFit, ImageShape } from './image.options.js';
+
+/**
+ * The base class used for constucting a fluent image custom element
+ * @public
+ */
+export class Image extends FASTElement {
+  /**
+   * Image layout
+   *
+   * @public
+   * @remarks
+   * HTML attribute: block.
+   */
+  @attr({ mode: 'boolean' })
+  public block?: boolean;
+  /**
+   * Image border
+   *
+   * @public
+   * @remarks
+   * HTML attribute: border.
+   */
+  @attr({ mode: 'boolean' })
+  public border?: boolean;
+  /**
+   * Image shadow
+   *
+   * @public
+   * @remarks
+   * HTML attribute: shadow.
+   */
+  @attr({ mode: 'boolean' })
+  public shadow?: boolean;
+
+  /**
+   * Border Radius
+   *
+   * @public
+   * @remarks
+   * Degree of border roundness.
+   */
+  @attr({ attribute: 'border-radius' })
+  public borderRadius: ImageBorderRadius;
```

Comment:
What is the motivation for `borderRadius`? I don't see it in FUIR9 nor in the Figma spec.

Reply:
![image](https://user-images.githubusercontent.com/12279161/220759533-81f45dc6-6c8b-4d8b-80f0-9a1d034c82ef.png)

Reply:
I think the attribute here is probably `shape`: https://master--628d031b55e942004ac95df1.chromatic.com/?path=/docs/components-image--default#shape

Reply:
Shape variations: Square, Rounded or Circular. When Rounded, the corners can have radii of Small, Medium, Large or X-Large.

Reply:
Should the prescribed border-radius options be CSS only and 100% at the whim of the consumer?

Reply:
The spec reference is here: https://github.com/microsoft/fluentui/blob/master/packages/react-components/react-image/docs/Spec.md#api-proposal

I'm not sure what the above is referring to but there isn't an API currently for it. It's possible the intent is guidance on what would be valid tokens to use for "rounded". An example, in Avatar I believe that the radius for the "square" variant changes based on the size. It's possible that was the intent, but we don't really have a way to do that with images because their size can be set a variety of ways that we don't have exposure to. It seems like this was the "intent" but it makes sense to me why this wouldn't actually be something we can implement.

A few suggestions:
1) We can research the history via PR's going into image for insight here.
2) We can reach out to the owners of Image to get their thoughts
3) We can raise the issue w/ design around the confusion with the spec to get their insight into intent.

For moving forward, my suggestion would be to remove this given that this doesn't exist yet in React. We can always add it later if we find that this is indeed an intended attribute/prop. Without knowing, it's typically better to start narrow and broaden the API. While we're in alpha, we'll be "breaking" folks who implement relying on this. While that's "doable" in alpha, we should avoid it if we can.

Reply:
Thanks for this helpful bit of feedback! I'll remove the border radius attribute.